### ISSUE TITLE
NTR: DHCA (dihydrocholestanoic acid) high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8837]
New term request DHCA (dihydrocholestanoic acid) high in urine (elthat_101118024045):

**Comment by pnrobinson** [comment]
@MickeySegal
the closest I can find is https://www.ebi.ac.uk/ols/ontologies/chebi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_131648

Please provide context

**Comment by MickeySegal** [comment]
The spelling "dihydrocholestanoic" seems to have been an error.  It is now corrected to dihydroxycholestanoic in our original source  https://www.ncbi.nlm.nih.gov/books/NBK1448/ but is still wrong in another article by one of the same authors https://clinicalgate.com/peroxisomal-disorders/.  

So we should keep it, but with corrected spelling.

The same applies to our finding now corrected to "DHCA (dihydroxycholestanoic acid) high in blood" (segal_070920164116).

**Comment by pnrobinson** [comment]
@MickeySegal 
I think it must be DHCA and THCA as in this article?
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5056113/

**Comment by MickeySegal** [comment]
Yes, the DHCA metabolite.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001006
- label: Elevated urine dihydroxycholestanoic acid level
- parents: [Abnormal urine carbohydrate level (HP:0031979)](https://hpo.jax.org/app/browse/term/HP:0031979)
- pmid: PMID:27647924
- def: The amount of dihydroxycholestanoic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8837
- synonyms: Elevated urine DHCA level(NONE)
